ui-charts/src/spaceTimeChart/components/PathLayer.tsx

```diff
@@ -336,8 +345,48 @@ export const PathLayer = ({
     [path]
   );
 
+  const drawBorder = useCallback<DrawingFunction>(
+    (ctx, stcContext) => {
+      if (!border) return;
+      const borderWidth = border.width || 1;
+      const mainPathStyle = STYLES[level];
+      const totalPathWidth = border.offset * 2 + mainPathStyle.width;
+
+      const segments = getPathSegments(stcContext);
+      ctx.save();
+      const drawSegments = (lineWidth: number, borderColor = border.color) => {
+        ctx.strokeStyle = borderColor;
+        ctx.lineWidth = lineWidth;
+        ctx.beginPath();
+        segments.forEach(({ x, y }, i) => {
+          if (x === segments[i - 1]?.x && y === segments[i - 1]?.y) return;
+          if (i === 0) {
+            ctx.moveTo(x, y);
+          } else if (i === segments.length - 1) {
+            ctx.lineTo(x - border.offset / 2, y);
```

Comment:
Why the `- border.offset / 2` here? It seems like the spacing on the right is smaller than `offset` with this adjustment.

Reply:
It was unnecessary, i wasn't seeing the end of the line well because of the display device so i tried unusefully to see something at the end i guess... so => removed ! thanks